### PR TITLE
Documentation update for webdriver protocol

### PR DESCRIPTION
**Pull request by praveendvd**
## Proposed changes

Fix for Protocol Documentation regarding webdriver findelement, getactive element etc are unclear #7113

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by praveendvd** [comment]
@christian-bromann  could you retrigger this , seems like a one time failure

File: packages/wdio-protocols/protocols/webdriver.json [hunk 1]
```diff
@@ -374,8 +374,16 @@
   "/session/:sessionId/element": {
     "POST": {
       "command": "findElement",
-      "description": "The Find Element command is used to find an element in the current browsing context that can be used for future commands.",
+      "description": "The Find Element command is used to find an element in the current browsing context that can be used for future commands.This command returns JSON representation of the element that can be passed to $ command to transform the reference to an extended WebdriverIO element.",
```
Comment (christian-bromann) [inline packages/wdio-protocols/protocols/webdriver.json:377]:
```suggestion
      "description": "The Find Element command is used to find an element in the current browsing context that can be used for future commands. This command returns JSON representation of the element that can be passed to $ command to transform the reference to an extended WebdriverIO element.",
```

File: packages/wdio-protocols/protocols/webdriver.json [hunk 2]
```diff
@@ -397,7 +405,7 @@
   "/session/:sessionId/elements": {
     "POST": {
       "command": "findElements",
-      "description": "The Find Elements command is used to find elements in the current browsing context that can be used for future commands.",
+      "description": "The Find Elements command is used to find elements in the current browsing context that can be used for future commands.This command returns array of JSON representation of the elements that can be passed to $ command to transform the reference to an extended WebdriverIO element (See findElement).",
```
Comment (christian-bromann) [inline packages/wdio-protocols/protocols/webdriver.json:408]:
```suggestion
      "description": "The Find Elements command is used to find elements in the current browsing context that can be used for future commands. This command returns array of JSON representation of the elements that can be passed to $ command to transform the reference to an extended WebdriverIO element (See findElement).",
```

File: packages/wdio-protocols/protocols/webdriver.json [hunk 3]
```diff
@@ -420,7 +428,7 @@
   "/session/:sessionId/element/:elementId/element": {
     "POST": {
       "command": "findElementFromElement",
-      "description": "The Find Element From Element command is used to find an element from a web element in the current browsing context that can be used for future commands.",
+      "description": "The Find Element From Element command is used to find an element from a web element in the current browsing context that can be used for future commands.This command returns JSON representation of the element that can be passed to $ command to transform the reference to an extended WebdriverIO element (See findElement).",
```
Comment (christian-bromann) [inline packages/wdio-protocols/protocols/webdriver.json:431]:
```suggestion
      "description": "The Find Element From Element command is used to find an element from a web element in the current browsing context that can be used for future commands. This command returns JSON representation of the element that can be passed to $ command to transform the reference to an extended WebdriverIO element (See findElement).",
```

File: packages/wdio-protocols/protocols/webdriver.json [hunk 4]
```diff
@@ -447,7 +455,7 @@
   "/session/:sessionId/element/:elementId/elements": {
     "POST": {
       "command": "findElementsFromElement",
-      "description": "The Find Elements From Element command is used to find elements from a web element in the current browsing context that can be used for future commands.",
+      "description": "The Find Elements From Element command is used to find elements from a web element in the current browsing context that can be used for future commands.This command returns array of JSON representation of the elements that can be passed to $ command to transform the reference to an extended WebdriverIO element (See findElement).",
```
Comment (christian-bromann) [inline packages/wdio-protocols/protocols/webdriver.json:458]:
```suggestion
      "description": "The Find Elements From Element command is used to find elements from a web element in the current browsing context that can be used for future commands. This command returns array of JSON representation of the elements that can be passed to $ command to transform the reference to an extended WebdriverIO element (See findElement).",
```

File: packages/wdio-protocols/protocols/webdriver.json [hunk 5]
```diff
@@ -474,7 +482,7 @@
   "/session/:sessionId/element/active": {
     "GET": {
       "command": "getActiveElement",
-      "description": "Get Active Element returns the active element of the current browsing context’s document element.",
+      "description": "Get Active Element returns the active element of the current browsing context’s document element.This command returns JSON representation of the element that can be passed to $ command to transform the reference to an extended WebdriverIO element (See findElement).",
```
Comment (christian-bromann) [inline packages/wdio-protocols/protocols/webdriver.json:485]:
```suggestion
      "description": "Get Active Element returns the active element of the current browsing context’s document element. This command returns JSON representation of the element that can be passed to $ command to transform the reference to an extended WebdriverIO element (See findElement).",
```

**Review by christian-bromann — APPROVED** [review]
Thanks a lot, LGTM 👍 

Mind accepting these small suggestions?

**Pull request merged**